Add acceptance test for create-read events.

igkon opened the pull request:
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Igor <hokletka@gmail.com>

Add robotest for checking read operation. It creates 5 events and then reads them according to scenario.

Commit:
Add acceptance test for create-read events.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Igor <hokletka@gmail.com>

Add robotest for checking read operation. It creates 5 events and then reads them according to scenario.

Commit:
Fix to read all events correctly

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Igor <hokletka@gmail.com>

akurilov commented:
Please merge master branch into yours to make it running the tests on GL CI.

akurilov reviewed (changes requested):
Please merge master branch into yours to make it running the tests on GL CI.

Commit:
Merge branch 'master' of https://github.com/emc-mongoose/mongoose-storage-driver-pravega into issue-76-acceptanceTestForReadEvents

akurilov commented on src/test/robot/api/storage/data/create_and_read_events_test.js:
```diff
@@ -0,0 +1,36 @@
+	        }
+	    }
+	})
+	.run();
```
shows missing newline

akurilov commented on src/test/robot/api/storage/create_and_read_events.robot:
```diff
@@ -0,0 +1,90 @@
+    Validate Metrics Total Log File  ${step_id}  READ  5  0  5242880
+
+*** Keyword ***
+Execute Mongoose Scenario
```
the keywords here and below are copy-paste product
please consider refactoring to reuse the keywords as was done in s3 driver, for example:
https://github.com/emc-mongoose/mongoose-storage-driver-s3/blob/master/src/test/robot/lib/MongooseContainer.robot
https://github.com/emc-mongoose/mongoose-storage-driver-s3/blob/master/src/test/robot/lib/Validation.py

otherwise, please put all tests in one robot file (easier and faster way)

Commit:
Merge branch 'master' of https://github.com/emc-mongoose/mongoose-storage-driver-pravega into issue-76-acceptanceTestForReadEvents

Commit:
Merge branch 'master' of https://github.com/emc-mongoose/mongoose-storage-driver-pravega into issue-76-acceptanceTestForReadEvents

Commit:
Create joint robotest file containing all tests

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Igor <hokletka@gmail.com>

Create joint robotest file and move mongoose keys of each test into scenarios.

Commit:
Merge branch 'master' of https://github.com/emc-mongoose/mongoose-storage-driver-pravega into issue-76-acceptanceTestForReadEvents

Commit:
Change gitlab-ci.yml file

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Igor <hokletka@gmail.com>

Read doesn't work in the create-read test.

akurilov commented:
Moved to another PR
